printerObjects.ps1

```diff
@@ -0,0 +1,19 @@
+class Snapshot {
+    #snapshot properties
+    [string]$Server;
+    [string]$Printer;
+    [int]$Action
+    [string]$Flag
+    #use class for variable validation and error control 
+    Snapshot(
+    [string]$s,
+    [string]$p,
```

Comment:
This needs review, the psscript tester should find any errors in the class definitions, it will require testing to determine what else needs to be caught.